how-to.md

```diff
@@ -0,0 +1,194 @@
+# How to WP-CLI
```

Comment:
I see that you've put everything into one single document.

I think it would be preferable to put the individual how-tos into separate pages, so that we can make use of the right-hand sidebar.

Reply:
OK perfect. 
I have just one question. Shall I keep the how-to page and create separate pages for each tutorial then reference each tutorial inside the how-to page?